packages/vue/src/components/organisms/SfProductCard/SfProductCard.stories.js

```diff
@@ -83,13 +83,44 @@ storiesOf("Organisms|ProductCard", module)
       alert(label) {
         alert(label);
       },
+      handleSelectedColor(color) {
+        this.colors.map((el) => {
+          if (el.label === color.label) {
+            el.selected = !el.selected;
+          } else {
+            el.selected = false;
+          }
+        });
+      },
```

Comment:
I wonder if it's necessary here? You have already handled selecting color inside SfProductCard component. Here I would add sth like this: 
``` 
alertColor(event) {alert(event.value);}
```
and then inside template:
```
 @click:colors="alertColor($event)"
```
To show that emitting event works properly like with other events. What do you think?
By the way there is a bug in storybook in emitting add to cart event, but I will handle that :)

Reply:
I think it's nice to have it. Thanks to it you can see how it really works - you can change color - and see a way how to implement it so one step easier :)